conf/machine/include/imx-base.inc

```diff
@@ -332,6 +332,10 @@ IMAGE_BOOT_FILES ?= " \
     ${@bb.utils.contains('MACHINE_FEATURES', 'optee', '${OPTEE_BOOT_IMAGE}', '', d)} \
 "
 
+IMAGE_INSTALL_append = " \
+    ${@bb.utils.contains('COMBINED_FEATURES', 'jailhouse', 'jailhouse', '', d)} \
+"
```

Comment:
`IMAGE_INSTALL` in machine files is usually not such a great idea to have (even if it is conditional). Can this be moved out?

Reply:
I'm not sure where is the best place ..but as you already noticed, it is conditional.

Reply:
> I'm not sure where is the best place ..but as you already noticed, it is conditional.

Conditional is exactly the problematic part... What if I want to have only one image recipe with `jailhouse` included? If I set a feature - all my image recipes would have `jailhouse` which would not be my intention.

I guess this line could be completely removed, and let developer to decide which image needs `jailhouse`.

Reply:
It can go to `MACHINE_EXTRA_RRECOMMENDS` so this is included on the image but can be filtered if needed.

Reply:
Simply, like `MACHINE_EXTRA_RRECOMMENDS = " jailhouse"` ?

Reply:
or `MACHINE_EXTRA_RRECOMMENDS_append_mx8 = " jailhouse"`

Reply:
no, it needs to be conditional to the combined features.